src/Form/molecules/ActionButtons.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styled from 'styled-components';
+import IconButton, { IconButtonData } from '../atoms/IconButton';
+
+const StyledIconButton = styled(IconButton)``;
+
+const Container = styled.div`
+  display: flex;
+  ${StyledIconButton} {
+    margin-left: 15px;
+  }
+  ${StyledIconButton}:first-child {
+    margin-left: 0px;
+  }
+`;
```

Comment:
Currently ActionButtons keeps them in the same row, but I'm not sure if it would be something desirable to also be on top of each other. One Idea to consider is that It could have a directionProp.

Reply:
For the foreseeable future we only need them in a row as they are currently. There may be a case for vertical stacked in the future but it's not an essential requirement right now.